sdk/storage/storage-blob/samples/typescript/basic.ts

```diff
@@ -18,28 +12,21 @@ async function main() {
   // Use SharedKeyCredential with storage account and account key
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
 
```

Comment:
Add comments to mention it's only available for Node.js. Same for all other samples